### PR TITLE
Fix rounding error in cart

### PR DESCRIPTION
**Pull request by tutturen**
* Fixed bug in `getPrice` function
* Added tests

### CODE REVIEW

**Comment by tutturen** [comment]
Closes #41

File: package.json
Comment (tobinus) [inline package.json]: Looks like this should wait until #25 is merged..?

Comment (tobinus) [reply package.json]: It's a bit difficult to review this before #25 is merged, because this is now equal to #25 plus a bit more.

**Review by tobinus — APPROVED** [review]
I liked that you made tests for this, made it way easier to try out other approaches.

**Pull request merged**
